Documentation/cli-installation-scenarios.md

```diff
@@ -242,8 +262,16 @@ OSX/Linux:
 
 #### Windows obtain one-liner example 
 
+CMD:
+
+```
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex}"
+```
+
+PowerShell:
+
 ```
-@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1'))}"
+iwr https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1 | iex
```

Comment:
@jasonwilliams200OK, thanks for the follow-up.  What do you thinking about something like [these changes](https://github.com/MichaelSimons/cli/commit/7629dc77fd5f132be2db5fd8b71839078499e510) instead?  I added logic to throw and emit better diagnostic information in the case when the request fails.  I propose this as I noticed the error experience is less than useful when for example you don't have internet access.  I also addressed the variable not set issues.